### ISSUE TITLE
[v2]  iOS nativeEventListener event name returns undefined

### ISSUE THREAD

**Issue by yusufyildirim** [wix/react-native-navigation#3255]
### Issue Description
I registered a nativeEventListener for detecting bottomTab changes. Everything was looking good, events were coming. Then I realized, event name is undefined but params object is coming as expected.

### Steps to Reproduce / Code Snippets / Screenshots
Try this;
```
function commandListener(name, params) => { console.log(name, params) };
Navigation.events().registerNativeEventListener(commandListener);
```

---
### Environment
* React Native Navigation version: v2@alpha
* React Native version: 0.55.4
* Platform(s) (iOS, Android, or both?): iOS (Didn't test Android but I looked at code and it seem okay)
* Device info (Simulator/Device? OS version? Debug/Release?): Simulator / iOS 11.3

**Comment by yusufyildirim** [comment]
Here is the pr to fix problem and detailed explanation #3254

**Comment by yogevbd** [comment]
PR approved, Thanks!
